app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -44,13 +43,18 @@ import org.kiwix.kiwixmobile.kiwixActivityComponent
 import org.kiwix.kiwixmobile.settings.KiwixSettingsActivity
 import org.kiwix.kiwixmobile.webserver.ZimHostActivity
 
-class KiwixMainActivity : CoreMainActivity() {
-  private lateinit var navController: NavController
-  private lateinit var appBarConfiguration: AppBarConfiguration
+const val PAGE_URL_KEY = "pageUrl"
+const val ZIM_FILE_URI_KEY = "zimFileUri"
 
+class KiwixMainActivity : CoreMainActivity() {
   private var actionMode: ActionMode? = null
 
-  val cachedComponent by lazy { kiwixActivityComponent }
+  override val cachedComponent by lazy { kiwixActivityComponent }
+  override val navController by lazy { findNavController(R.id.nav_host_fragment) }
+  override val drawerContainerLayout by lazy { navigation_container!! }
+  override val drawerNavView by lazy { drawer_nav_view!! }
+  override val bookmarksFragmentResId: Int = R.id.bookmarksFragment
```

Comment:
pretty sure these have the same values in both subclasses

Reply:
They reference their separate view ids from different modules. I don't think it works to pull them up to core even though they have the same id name. 

### In CoreMain:
![image](https://user-images.githubusercontent.com/22193232/89288298-b1192200-d655-11ea-878b-5154bdd38849.png)

Reply:
I meant the fragments ids but I guess they reference totally separate ids from different graphs?

Reply:
ah, yes, they do.